SetupDataPkg/ConfApp/SetupConf.c

```diff
@@ -925,7 +1010,8 @@ CreateXmlStringFromCurrentSettings (
     Status = Base64Encode (Data, DataSize, EncodedBuffer, &EncodedSize);
     if (EFI_ERROR (Status)) {
       DEBUG ((DEBUG_ERROR, "Failed to encode binary data into Base 64 format. Code = %r\n", Status));
-      return EFI_INVALID_PARAMETER;
+      Status = EFI_INVALID_PARAMETER;

```

Comment:
Should we be freeing data in this case?

Reply:
Yeah, going into EXIT should cover allocated buffers.

Reply:
I see it now